packages/ph-cli/src/commands/install.ts

```diff
@@ -0,0 +1,323 @@
+import path, { dirname } from "node:path";
+import fs from "node:fs";
+import { execSync } from "node:child_process";
+import { homedir } from "node:os";
+import { Command } from "commander";
+
+import { CommandActionType } from "../types.js";
+
+type PowerhouseConfig = {
```

Comment:
We should update the type on @powerhousedao/config and import it here

Reply:
added fix here: [e797588](https://github.com/powerhouse-inc/powerhouse/pull/831/commits/e7975889b22389348f6702b076e8ad3e04513463)